packer/grapl-service/setup_nomad_consul.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
```

Comment:
`set -euo pipefail` will be better (here, and in probably every Bash script you write). 

Among other things, that will fail if you forget to pass in one of the required environment variables.

Reply:
yep, this is a leftover from the hashicorp template